packages/cli/src/ceramic-daemon.ts

```diff
@@ -137,8 +139,9 @@ class CeramicDaemon {
   async listen(): Promise<void> {
     return new Promise<void>((resolve) => {
       const port = this.opts.port || DEFAULT_PORT
-      this.server = this.app.listen(port, () => {
-        this.diagnosticsLogger.imp('Ceramic API running on port ' + port)
+      const hostname = this.opts.hostname || DEFAULT_HOSTNAME
```

Comment:
By setting this to `localhost` by default, now are we going to start rejecting connections not from localhost by default?